lib/fastlane/plugin/ci_changelog/helper/ci_changelog_helper.rb

```diff
@@ -17,8 +17,10 @@ def self.dump_jenkin_commits(body)
         # TODO: It must use reverse_each to correct the changelog
         commit = json['changeSet']['items'].each_with_object([]) do |item, obj|
           obj.push({
+            id: item['commitId'],
             date: item['date'],
-            message: item['msg'],
+            title: item['msg'],
```

Comment:
It's better to keep `message` key and use `body` for comments.

Reply:
The `comment` in Jenkins API actually includes both the title and the body (aka the full commit message), so I think using `message` should be more accurate.